### ISSUE TITLE
Selinux flag giving false positive 

### ISSUE THREAD

**Issue by Man3006** [scottyab/rootbeer#177]
Selinux flag is indicating device as rooted though it is not.
tested on other device Samsung galaxy j7 prime also gave false positive with selinix flag tag. even though other devices i checked dint gave false positive with this tag. 
But i thik this tag should not be used.. giving false result in so many devices don't know why @
`

**Comment by fnazal18** [comment]
It's pretty common to have some false positives while checking SELinux enforcing through properties. I did it long time ago and I came up with testing it by using the libselinux in a shared library. It's slightly better 'cause if the mode can't be read it's assumed that the device is in enforcing mode (EACCES), and if it's not the case (the mode can be retrieved), we can check the value returned (either 0 or any value greater than 0) and return the boolean or whatever you want. 

Still, it's a good way to check if the device may be rooted. So, I think it just need some adjustments.
